Code/Tools/TestImpactFramework/Platform/Windows/PAL_windows.cmake

```diff
@@ -6,4 +6,4 @@
 #
 #
 
-set(PAL_TRAIT_TEST_IMPACT_FRAMEWORK_SUPPORTED TRUE)
+set(PAL_TRAIT_TEST_IMPACT_FRAMEWORK_SUPPORTED FALSE)
```

Comment:
Note to reviewers, We've already started  deprecating it and no longer use in dev windows builds, but this turns off more pieces of it.